snapshot.yaml

```diff
@@ -10,16 +10,21 @@ packages:
   - HPDF-1.5.1
   - odd-jobs-0.2.2
   - patrol-0.0.1
-  - persistent-2.11.0.2
-  - persistent-template-2.9.1.0
-  - persistent-mongoDB-2.11.0.0
-  - persistent-postgresql-2.11.0.1
   - prolude-0.0.0.17
   - servant-lucid-0.9.0.2
   - servant-serf-0.1.0
   - servant-static-th-1.0.0.0
   - timing-convenience-0.1
 
+  # https://github.com/yesodweb/persistent/pull/1209
+  - git: https://github.com/yesodweb/persistent
+    commit: ec5f9ff9373f511d520bc8dd33cfab09db50cab9
+    subdirs:
+     - persistent
+     - persistent-mongoDB
+     - persistent-postgresql
+     - persistent-template
```

Comment:
It's fine to depend on the GitHub versions of these, but I think `persistent` is the only important one.

Reply:
That's right, I can remove the others. This is just an artifact of testing with the non-backported persistent that meant I had to list all of them.

I'll leave them just in case since when he finishes the release soon I'll be updating this again and they'll essentially be the same revision.